src/phoenix/trace/llama_index/callback.py

```diff
@@ -285,31 +299,47 @@ def _add_to_tracer(
     parent_child_id_stack: List[Tuple[Optional[SpanID], CBEventID]] = [
         (None, root_event_id) for root_event_id in trace_map["root"]
     ]
+    span_events: List[SpanEvent] = []
     while parent_child_id_stack:
         parent_span_id, event_id = parent_child_id_stack.pop()
         event_data = event_id_to_event_data[event_id]
         start_event = event_data["start_event"]
-        start_time_tz_naive = datetime.strptime(start_event.time, TIMESTAMP_FORMAT)
-        start_time_tz_aware = start_time_tz_naive.replace(tzinfo=_LOCAL_TZINFO)
-        end_event = event_data["end_event"]
-        end_time_tz_naive = datetime.strptime(end_event.time, TIMESTAMP_FORMAT)
-        end_time_tz_aware = end_time_tz_naive.replace(tzinfo=_LOCAL_TZINFO)
-        name = event_data["name"]
         event_type = event_data["event_type"]
+        if event_type is CBEventType.EXCEPTION:
+            if (
+                not start_event.payload
+                or (error := start_event.payload.get(EventPayload.EXCEPTION)) is None
+            ):
+                continue
+            span_events.append(
```

Comment:
Just for clarity, probably better give it its own list.
```suggestion
            span_exceptions.append(
```

Reply:
addressed in [4dd7df3](https://github.com/Arize-ai/phoenix/pull/1449/commits/4dd7df31606bcffbc8c7c42456646624351b44c2)